.github/workflows/ci.yml

```diff
@@ -179,6 +179,19 @@ jobs:
     - name: make test
       run: make test HARNESS_JOBS=${HARNESS_JOBS:-4}
 
+  fips_and_ktls:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: modprobe tls
+      run: sudo modprobe tls
+    - name: config
+      run: ./config --banner=Configured --strict-warnings enable-ktls enable-fips && perl configdata.pm --dump
```

Comment:
We already have enable-ktls and enable-fips defined for the `enable-non-default_options` build. Why did that build not detect the problem?

Reply:
I think we don't have kernel 5.11+ in our CI disposal

Reply:
Not sure how does it interfere with `no-ec`, BTW

Reply:
If I googled it correctly, Ubuntu-latest has just 5.8 kernel

Reply:
The way I've probed the VMs information is by adding steps in the CI which print what I want.  It's tedious but works.